xrpl4j-model/src/main/java/org/xrpl/xrpl4j/model/client/accounts/NfTokenObject.java

```diff
@@ -0,0 +1,42 @@
+package org.xrpl.xrpl4j.model.client.accounts;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import org.immutables.value.Value;
+
+import java.util.Optional;
+
+/**
+ * Structure of an NFToken stored on the ledger.
+ */
+@Value.Immutable
+@JsonSerialize(as = ImmutableNfTokenObject.class)
+@JsonDeserialize(as = ImmutableNfTokenObject.class)
+public interface NfTokenObject {
```

Comment:
This class needs to go in the `@JsonSubTypes` annotation on `LedgerObject`.

Reply:
This is  just a helper object. I think NfTokenOfferObject is the one that would go on this list. ref: https://xrpl.org/nftokenoffer.html